test/downstream/nllsopt.jl

```diff
@@ -0,0 +1,29 @@
+using NonlinearSolve, Optimization, OptimizationNLopt, ForwardDiff
+import FastLevenbergMarquardt, LeastSquaresOptim
```

Comment:
These downstream deps would be required. But just remove `FastLevenbergMarquardt` and `LeastSquaresOptim`?